.github/workflows/releaser.yml

```diff
@@ -8,31 +8,23 @@ on:
 
 jobs:
 
-  # awx compatible binary with correct version of gcc and clang
-  binary_centos7_amd64:
```

Comment:
drop awx centos7 builds as no longer required (polycli handles key creation now)

Reply:
This is great. Simplifies things